config.php

```diff
@@ -37,15 +37,17 @@
 			'action' => 'comment',
 			'label' => 'question',
 			'comment' => <<<COMMENT
-_This is an automated comment, triggered by adding the label `question`._
-			
-Please note, that the GitHub Issue Tracker is for bug reports and feature requests only.
+Thank you for your question in our issue tracker. In order for this issue tracker to be effective,
+we ask our community to use our online resources specifically designed for support:
```

Comment:
the intention was to formulate something that 'we have setup something specifically for support'

I dont like the word setup. 'Design' is better, perhabs there are words thinkable that are even better.